### DISCUSSION TITLE
MAMP Cask

### DISCUSSION THREAD

**Discussion by dtmpsc** [Homebrew/discussions#5523] [Casks]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.3.10
ORIGIN: https://github.com/Homebrew/brew
HEAD: c3ae0327c01aab461531bfc0fcdfb4dd3a779205
Last commit: 8 days ago
Core tap JSON: 26 Jul 21:13 UTC
Core cask tap JSON: 26 Jul 21:13 UTC
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 8
Homebrew Ruby: 3.3.3 => /opt/homebrew/Library/Homebrew/vendor/portable-ruby/3.3.3/bin/ruby
CPU: octa-core 64-bit arm_firestorm_icestorm
Clang: 15.0.0 build 1500
Git: 2.39.3 => /Library/Developer/CommandLineTools/usr/bin/git
Curl: 8.6.0 => /usr/bin/curl
macOS: 14.5-arm64
CLT: 15.3.0.0.1.1708646388
Xcode: 15.4
Rosetta 2: false
```

### Output of `brew doctor`

```shell
Your system is ready to brew.
```

### Description of issue

brew MAMP install fails on error SHA256 hash mismatch. 
MAMP package download has correct SHA512 hash listed on https://www.mamp.info/en/downloads/

### Relevant casks

MAMP

**Comment by p-linnane** [comment]
There is an open PR for this: https://github.com/Homebrew/homebrew-cask/pull/180318. We are waiting for someone to contact upstream and confirm the change was intentional, and not malicious.
